activiti-cloud-notifications-graphql-service/services/security/src/main/java/org/activiti/cloud/services/notifications/qraphql/ws/security/WebSocketMessageBrokerSecurityAutoConfiguration.java

```diff
@@ -34,12 +32,6 @@
     })
     public static class DefaultWebSocketMessageBrokerSecurityConfiguration {
 
-        @Bean

```

Comment:
If I leave the configuration in this way the test doesn't find the beans configured in AbstractSecurityWebSocketMessageBrokerConfigurer, so I transformed it in a class annotated with @Configuration.

Reply:
it matches what the doc says https://docs.spring.io/spring-security/site/docs/current/reference/html5/#websocket-configuration

Reply:
This change looks good to me..